### DISCUSSION TITLE
familyの見直し

### DISCUSSION THREAD

**Discussion by noborus** [pgsql-jp/jpug-doc#3119] [NGとする語]
### 重要度

低

### 問題点

#2617 により、ファミリー→ファミリにする予定だったが、ファミリの方が1箇所しか使われておらず、ファミリーの方が多く、
直すべきか再考が必要。

### 背景

familyが使用されているのは、以下です。

* operator family 
* address's family 
* OSのfamily
* CPUのfamily

operator family は「演算子族」と訳しているので、btree.sgmlの「ファミリー」は「族」のように修正した方がよさそう。
address's family はアドレスファミリーで、他でも使われていて妥当。アドレスファミリの場合もある。
OSとCPUのfamilyは日本語だと「系」とか意訳されている場合が多い。

### 解決方法

ネットワークアドレス以外でファミリーを使用しないように修正するのが良いのではないか。

ネットワークアドレスでも「ファミリー」のままで良いのではないか。
ファミリーは専門用語よりも一般に使用されている場合が多く、「ファミリ」よりも「ファミリー」の方が馴染みがある。

### 貢献者として記載可否

以前の記載名を使用

**Comment by koizumistr** [comment]
> ネットワークアドレスでも「ファミリー」のままで良いのではないか。

Geminiさんも「アドレスファミリー」と「アドレスファミリ」のどちらが一般的か尋ねたところ
> JIS X 0007（情報処理用語）」での表記: この規格では「アドレスファミリー」と表記されています。（これはハルシネーションっぽいですけど）
> 一般的には、「アドレスファミリー」と表記するのが適切です。
> 技術的な文書や専門的な場面では、「アドレスファミリー」を使うのが望ましい。

と言ってますので、それでいいと思います。

ちなみにMSの文書では矛盾してますね。
https://learn.microsoft.com/ja-jp/windows-hardware/drivers/network/address-families

> ネットワークアドレス以外でファミリーを使用しないように修正するのが良いのではないか。

こちらについても同意です。
